FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -18,7 +18,63 @@ public final class RemoteFeedLoader: FeedLoader {
 		self.client = client
 	}
 
+	private var OK_200: Int { return 200 }
```

Comment:
The `OK_200` instance property can be static instead, since there's no need for an instance of `RemoteFeedLoader` to retrieve a constant value. It would be more suitable to have an instance property if the value varies per instance.
```suggestion
	private static var OK_200: Int { return 200 }
```